src/Printer.php

```diff
@@ -77,28 +85,28 @@ public function trace(Event $event): void
                 $this->status = TestStatus::Risky;
             }
 
-            $this->trace = new Trace($event->message(), $event->test()->file(), $event->test()->line());
+            $this->uniqueTraces[] = new Trace(TestStatus::Risky, $event->message(), $event->test()->file(), $event->test()->line());
```

Comment:
Why isn't this calling `addUniqueTrace`?

Reply:
Risky is a test status (rather than issue status) and it's final. There can't really be dupes.

Reply:
There're 3 options:
1. Leave it this way.
1. Leave it this way, with an explanatory comment.
1. Use  `addUniqueTrace()` for coherent style.

I'm happy with the one you prefer.

Reply:
2

Reply:
Do you prefer I close conversations to signal I've made changes or let you close them to signal OK?

Reply:
Don't mind. I've resolved some I know have been addressed.